libs/shared/ui/src/lib/components/banner/banner.tsx

```diff
@@ -33,21 +34,22 @@ export const Banner = forwardRef<HTMLDivElement, PropsWithChildren<BannerProps>>
   forwardedRef
 ) {
   return (
-    <div className={bannerVariants({ color })}>
-      <div className="flex h-10 items-center justify-center font-medium text-sm" ref={forwardedRef}>
-        {children}
-        {buttonLabel && (
-          <Button
-            style={ButtonStyle.RAISED}
-            size={ButtonSize.TINY}
-            className={buttonVariants({ color })}
-            iconRight={buttonIconRight}
-            onClick={onClickButton}
-          >
-            {buttonLabel}
-          </Button>
-        )}
-      </div>
+    <div
+      className={twMerge('flex h-10 items-center justify-center font-medium text-sm', bannerVariants({ color }))}
```

Comment:
move `'flex h-10 items-center justify-center font-medium text-sm'`  into `bannerVariants` so you can get rid of `twMerge`

Reply:
fixed https://github.com/Qovery/console/pull/856/commits/0f232ffdc01d3a3f5b5618f00a0be5989e20aacf